modules/core/04-channel/keeper/grpc_query_test.go

```diff
@@ -732,21 +748,14 @@ func (suite *KeeperTestSuite) TestQueryPacketCommitments() {
 			false,
 		},
 		{
-			"success, empty res",
+			"Channel not found",
```

Comment:
I believe the "success, empty res" test is not needed anymore, since an empty res was what we were getting when providing an invalid channel (see related issue's first post). I am not 100% sure about this so I'm open to corrections!

Reply:
Yes, this sounds correct to me.